### ISSUE TITLE
[2021 fall] CVPR 2021 Scene Text Telescope - Text-focused Scene Image Super-Resolution (20213228)

### ISSUE THREAD

**Issue by psh01087** [awesome-davian/awesome-reviews-kaist#144]

**Comment by 2na-97** [comment]
안녕하세요, 올려주신 자료 잘 읽어 보았습니다.  
개인적인 comment는 다음과 같습니다.  

**1. Related Works**
- 많은 예시를 들어주신 것은 좋으나 _논문제목 - 간단한 설명_ 형식으로 되어있다 보니 가독성이 조금 떨어지는 것 같습니다. 모든 논문을 간략하게 설명하기보다는 각 섹션별로 대표적인 method를 자세하게 설명해주시면 task에 대한 이해도 오를 수 있고, 가독성도 좋아질 것으로 생각됩니다.
- 그 외의 논문들은 밑에 정리하거나, 아니면 reference에 참고자료로만 올려주셔도 괜찮을 것 같습니다.

**2. Method**
- STN이나 TBSRN과 같은 네트워크에 대한 설명이 조금 더 있으면 좋을 것 같습니다.
- 각 모듈에 대한 설명 후 이미지 트레이닝 과정을 순서대로 한 번 더 정리해주시면 이해가 더 잘 될 것 같습니다.

**기타 사항**
- 이탤릭체 적용이 안되어 있는 부분이 있는 것 같은데, 한 번 확인해 주시면 좋을 것 같습니다.

감사합니다.

**Comment by shjung13** [comment]
안녕하세요, 좋은 리뷰 감사드립니다.
덕분에 흥미롭게 잘 읽은 것 같습니다.

1. Problem definition
- 간단한 motivation을 포함해서 설명을 이해는데 무리가 없었습니다. 혹시 가능하시다면 task의 입출력을 명시해주실 수 있을까요? 이쪽 분야를 잘 몰라서 처음에는 text localization도 수행하는 논문으로 이해했네요. Text가 crop된 LR이미지가 네트워크의 입력이고 그것에 해당하는 text 를 예측하는게 예측값인듯하네요. (가장 첫줄인 일상적인 풍경 이미지에서 글자를 인식하는 task에서 오해를 한 듯 합니다)

2. Related work
- 다른 리뷰어 분께서 말씀해주신대로, 다양한 work에 대해서 제시해주신것은 좋은 것 같습니다. 하지만 한두개의 대표적인 연구들에 대해서는 조금 더 구체적인 설명을 해주시면 해당 논문의 novelty에 대한 이해가 더 잘 될듯합니다.
- Idea에서 3번째 bullet point에 있는 weighted cross entropy loss는 각  character별 데이터셋에 등장하는 횟수가 다르기 때문에 이 frequency imbalance를 보정해주기 위해 하는걸까요? 헷갈릴 수 있는 문자들을 고려하기 위해 weighted cross entropy loss를 적용하는 부분이 잘 연결이 안되네요.

3. Method
- pixel-wise supervision module에 대한 1번째 설명에서 misalignment 문제가 있다고 하는데 혹시 어떤 문제인지 조금 더 구체적으로 설명해주실 수 있나요? (제가 reference된 논문에서 이해한게 맞다면 데이터 셋을 구성할 때 LR / HR pair를  camera의 focal length를 조절하여 구하게 되고 그때 생기는 misalignment인 듯한데 맞나요?)
- pixel-shuffling에 대해서 간략히 설명해주실 수 있나요? 혹은 내용이 길다면 인용을 달아주실 수 있나요?
- positional-aware module에서 text label 의 길이가 l로 고정되어있는듯한데, 가장 긴 text의 길이로 l을 설정 후에 l보다 길이가 작은 text의 경우 text예측이 끝나면 특정 토큰을 예측하는 방식인가요?

4. Result
- Ablation study와 Results on TextZoom에 대한 실험결과에서 CRNN혹은 다른 OCR모델의 예측값을 GT로 가정하고 accuracy를 측정한게 것 같은데 제가 이해한게 맞나요?

제 리뷰는 여기까지입니다.

고생하셨습니다, 감사합니다.

**Comment by mjbooo** [comment]
안녕하세요 AI대학원 석사과정 이민재(20214501) 입니다.
먼저 정성들인 논문 리뷰 감사드려요

related work에 관해서는 다른 분들과 마찬가지 의견입니다 . 모든 논문을 간략히 하기보다는, 다루시는 모델과 직접적으로 연결되는 것을 조금 더 구체적인 설명과 함께 (무엇이 다르고 어떻게 더 좋은지) 말씀해주시는 방향이 나을 것 같습니다. 

Method 부분에서 3가지 모듈의 하위 모듈들 (e.g., STN, TBSRN, pixel shuffle, ....) 각각에 대해 아키텍쳐와 '줄글로 풀어쓴' 역할이 덧붙여지면 훨씬 이해하기 쉬울 것이라고 느꼈습니다.

3가지 모듈들에 대해서도 마찬가지인데, 예컨대 position aware module에서 attention map으로 L1 loss를 구해 backprop 하는 것이 어떻게 'position aware' 역할을 할 수 있는지 조금 거시적으로 풀어서 쓴 설명이 있으면 훨씬 가독성이 높아질 것 같습니다.

감사합니다!
